pom.xml

```diff
@@ -25,7 +25,6 @@
             <name>OSS Snapshots</name>
             <url>https://oss.sonatype.org/content/repositories/snapshots</url>
         </repository>
-        <!-- Temporary - needed for old snapshots of ChannelfinderAPI -->
```

Comment:
Why remove this comment? It still applies.

Reply:
Well, if it still applies then it's not temporary?

Reply:
Let me explain.
The ChannelFinder API has changed. The only place where Maven can find the old versions is the deprecated EPICS repository on SourceForge.
As soon as the directoryService example has been updated to use the current API of ChannelFinder, the pointer to this deprecated repository should be removed.

Is that "temporary"? Do you have a better term?

Reply:
Then I think the comment should state what you just wrote. Plus the version numbers for the "old" and "current" ChannelFinder API.